feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -0,0 +1,185 @@
+        cx-icon {
+          color: $sapFioriLinkColor;
+        }
+        cx-icon.previous {
+          padding: 0 10px 0 0;
+        }
+        cx-icon.next {
+          padding: 0 0 0 10px;
+        }
+        button {
+          text-decoration: none;
+          color: $sapFioriLinkColor;
+        }
+        .cx-btn-previous {
+          span {
+            padding: 0 0 0 5px;
+          }
+        }
+        .cx-btn-next {
+          span {
+            padding: 0 5px 0 0;
+          }
+        }
+        div:first-child {
+          font-size: 0.875rem;
+          color: $sapFioriTextColor;
+          padding: 0 10px 0 10px;
+        }
+      }
+      .table-header-cell {
+        line-height: 40px;
+        margin: 0 15px 0 0;
+      }
+      .table-header-cell.sort {
+        cursor: pointer;
+        cx-icon {
+          margin: 0 10px;
+        }
+      }
+    }
+    .cx-header-actions.mobile {
+      flex-direction: column;
+    }
+  }
+
+  .cx-dialog-sub-header.tablet-mobile {
+    flex-direction: column;
+  }
+
+  .cx-dialog-body {
+    padding-top: 0;
+    max-height: calc(100vh - 260px);
+    overflow-y: auto;
+  }
+
+  .table {
+    thead {
+      tr {
+        border-top: 1px solid var(--cx-color-light);
+        height: 44px;
+        th {
+          color: $sapFioriTextColor !important;
+          text-transform: none !important;
+          font-weight: 700 !important;
+        }
+      }
+    }
+    tbody {
+      td {
+        font-size: 0.875rem;
+        color: $sapFioriTextColor;
+        .cx-header-text {
+          color: $sapFioriContent_LabelColor;
+          font-weight: 400;
+        }
+      }
+    }
+  }
+
+  .cx-btn-cell {
+    text-decoration: none;
+    color: $sapFioriLinkColor;
+    padding: 0;
+    border: 0;
+    min-height: 1.15rem;
+    width: fit-content;
+  }
+  .cx-avatar-cell {
+    width: 65px;
+    vertical-align: top;
+    .cx-avatar {
+      width: 44px;
+      height: 44px;
+      line-height: 44px;
+      border-radius: 50%;
+      color: #fff;
+      text-align: center;
+      background: #9259cd;
+      text-transform: uppercase;
+      margin: 2px 10px 2px 0;
+    }
+  }
+
+  .cx-cell-container {
+    display: flex;
+    align-items: center;
+    min-height: 44px;
+    padding: 5px 0;
+  }
+
+  .cx-action-color {
+    color: $sapFioriLinkColor;
+  }
+
+  .cx-multi-cell {
+    vertical-align: top;
+    .cx-cell-container {
+      flex-direction: column;
+      align-items: flex-start;
+      button {
+        min-width: 0;
+      }
+      cx-icon {
+        align-self: flex-start;
+      }
+    }
+  }
+
+  .cx-empty-state {
+    display: flex;
+    justify-content: center;
+    margin: 50px 0 0 0;
+  }
+  ng-select {
+    .ng-select-container {
+      border: 0 !important;
+      .ng-arrow-wrapper {
+        .ng-arrow {
+          border-color: $sapFioriLinkColor transparent transparent !important;
+        }
+      }
+    }
+  }
+  .ng-select-focused {
+    outline: none !important;
```

Comment:
Same comment as above. Can this be made more specific to avoid the use of the !important.